grails-app/assets/stylesheets/homepage.css

```
@@ -0,0 +1,134 @@
+body {
+    background-color: #0030B9;
+    margin: 0px;
+}
+
+html, body {
+    height: 100%;
+}
+
+.page-container {
+    display: flex;
+    align-items: center;
+}
+
+.left-bar {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+    width: 33%;
+}
+
+.asaas-logo {
+    width: 10.9375rem;
+    height: auto;
+    margin-left: 1.25rem;
+    margin-top: 0.9375rem;
+}
+
+.cellphone {
+    width: 25rem;
+    height: auto;
+    margin-left: 1.875rem;
+}
+
+.center {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100vh;
+    width: 33%;
+}
+
+.top-text {
+    margin-top: 8.125rem;
+    margin-bottom: 8.125rem;
```

```suggestion
    margin-block: 8.125rem;
```
Mas talvez assim não funcione por causa da flex-direction no container pai. Se isso aplicar margem dos lados, pode trocar por margin-inline que deve funcionar.